setup.py

```diff
@@ -242,7 +242,10 @@ def run(self):
     'packages'           : find_namespace_packages('src', include=['radical.*']),
     'package_dir'        : {'': 'src'},
     'scripts'            : [
-                            'bin/radical-pilot-agent',
+                            'bin/radical-pilot-agent_0',
```

Comment:
@andre-merzky there is an error during setup: `error: [Errno 2] No such file or directory: 'bin/radical-pilot-agent_0'`